core/templates/services/question-validation.service.spec.ts

```diff
@@ -28,36 +28,27 @@ import { StateEditorService } from
 /* eslint-enable max-len */
 // ^^^ This block is to be removed.
 
-require('domain/question/QuestionObjectFactory.ts');
-require('services/question-validation.service.ts');
+import { QuestionObjectFactory, Question } from
+  'domain/question/QuestionObjectFactory';
+import { QuestionValidationService } from
+  'services/question-validation.service';
+import { TestBed } from '@angular/core/testing';
```

Comment:
Move the test bed over top and add a line break and then import oppia dependencies.

Reply:
Done